pkgs/servers/photoprism/default.nix

```diff
@@ -62,6 +62,8 @@ stdenv.mkDerivation {
       --set PHOTOPRISM_RAWTHERAPEE_BIN ${rawtherapee}/bin/rawtherapee-cli \
       --set PHOTOPRISM_HEIFCONVERT_BIN ${libheif}/bin/heif-convert \
       --set PHOTOPRISM_FFMPEG_BIN ${ffmpeg}/bin/ffmpeg \
+      --set PHOTOPRISM_IMAGEMAGICK_BIN ${imagemagick}/bin/convert \
+      --set PHOTOPRISM_RSVGCONVERT_BIN ${librsvg}/bin/rsvg-convert \
```

Comment:
I'm not entirely sold that this should be part of the package update.

1.) Even if we decide to always add the dependencies imagemagick and librsvg this could be done in a separate PR - or maybe in a separate commit within this PR?
2.) I'm not entirely sold if it is really necessary to add those dependencies. As far as I can tell those are only used for converting vector graphics which I'd imagine is a rather niche application of photoprism.

Anyway, thank you for taking the time to update the package, appreciated! :+1:

Reply:
The functionality for both was newly introduced within this release. 

I agree that the svg converter is optional, especially since it is a non community-edition feature. Imagemagic, however, is also used to fix invalid jpgs that the golang library has trouble with. I feel that this is closer to a core functionality that should be enabled by default. In my personal library this affects quite a few files.

My suggestion would be: 
- Keep only imagemagic in this PR 
- Create a separate PR/discussion on how to handle such optional dependencies. I think similar concerns can be raised for rawtherapee, which is quite heavy and (as far as i understand it) primarily used as a fallback raw converter if darktable fails.  (We could either parameterize the package, or leave the binary configuration to the nixos module which can dynamically include the tools based on the settings)

Reply:
Wasn't aware that imagemagick was used this way :+1: Your suggestions sounds good, I guess I'd prefer if the module handles those runtime dependencies instead of a parameterized package. But this is for another PR :)